services/app-api/src/emailer/emails/newPackageCMSEmail.ts

```diff
@@ -25,12 +30,24 @@ export const newPackageCMSEmail = async (
         pkg,
         stateAnalystsEmails
     )
+    const combinedProgramIDs = findAllPackageProgramIds(pkg)
+    //Get program data from combined program ids
+    const programs = findPrograms(pkg.stateCode, combinedProgramIDs)
```

Comment:
Sorry for all the churn, but I actually think that our call to findPrograms specifically should still be in the resolver. I was thinking that we would just grab a list of all the programs for the given state and pass that into the emailer. Oner day that lookup will likely require a database connection and I don't want to have to pass that into the emailer.